proto/orijtech/cosmosloadtester/v1/loadtest_service.proto

```diff
@@ -96,4 +96,47 @@ message RunLoadtestResponse {
   // The rate at which data was transmitted in transactions (bytes/sec).
   // Corresponds to avg_data_rate in tm-load-test.
   double avg_bytes_per_second = 5;
+
+  // The respective points per second from 0 until the request's max_time.
+  repeated PerSecond per_sec = 6;
```

Comment:
@odeke-em in chat, you mentioned:
> about the slice: we might have scenarios in which there is more than 1 connection so safe to return it as a slice. For starters let’s let it display stats per transactor. Once the app is being used we shall have breathing space to display an aggregation

This proto definition only returns aggregate values, while the custom tm-load-test returns a `[]*ProcessedStats`, with an entry for each transactor:

https://github.com/orijtech/tm-load-test/blob/d37154798c88c311e880eb23ec799d09e04cb44b/pkg/loadtest/loadtest.go#L15

Should the proto be changed or should the implementation aggregate the stats?